code/modules/mob/living/carbon/xenomorph/castes/widow/castedatum_widow.dm

```diff
@@ -0,0 +1,178 @@
+/datum/xeno_caste/widow
+	caste_name = "Widow"
+	display_name = "Widow"
+	upgrade_name = ""
+	caste_desc = " A spider xenomoroph, horrible"
+	caste_type_path = /mob/living/carbon/xenomorph/widow
+
+	tier = XENO_TIER_THREE
+	upgrade = XENO_UPGRADE_BASETYPE
+	wound_type = "widow"
+
+	// *** Melee Attacks *** //
+	melee_damage = 7
```

Comment:
Drone melee_damage from young to ancient is 16/16/18/18
Widow melee_damage from young to ancient is 7/7/12/15

Leedsifer | Lloyd Hext & SA-X, our funny headmin, recommends 7/7/12/15 for spiderling, not widow. A T3 should not be weaker than a drone. 
![image](https://user-images.githubusercontent.com/6610922/189552041-970d7746-77e1-4848-bcca-57a8fe5936e3.png)